ui/src/views/DashboardView/DashboardView.tsx

```diff
@@ -86,58 +78,55 @@ const DashboardView = () => {
 
   return (
     <List
+      title={
+        <>
+          Zaaktypen&nbsp;
+          {isLoading && <Outline.ArrowPathIcon spin={true} aria-label={'Bezig met laden'} />}
+        </>
+      }
+      formProps={{
+        autoComplete: 'off',
+        fields: [
+          {
+            disabled: loadingCatalogi,
+            defaultValue: 'all',
+            label: 'Selecteer catalogus',
+            options: selectOptions,
+            value: selectedCatalogus,
+            onChange: onCatalogusChange,
+          },
+          {
+            label: 'Filter resultaten',
+            placeholder: 'ZAAKTYPE-XXXX-XXXXXXXXXX',
+            size: 24,
+            value: query,
+            onChange: (e: React.ChangeEvent<HTMLInputElement>) => setQuery(e.target.value),
+          },
+        ],
+        showActions: false,
+      }}
       fields={fields}
       loading={isLoading}
-      results={filteredZaaktypen as unknown as AttributeData<Attribute>[]}
       sort={true}
       onClick={(_: React.MouseEvent, attributeData: AttributeData) => {
```

Comment:
When I click on a zaaktype link, I initially see this: (for very short)
![Screenshot from 2024-04-09 16-32-13](https://github.com/maykinmedia/open-zaaktypebeheer/assets/19154114/dd927e22-7b01-4dba-8a2f-f77788462745)
And then I go to the zaaktype detail page